catboost/python-package/catboost/core.py

```diff
@@ -84,6 +85,21 @@ def get_catboost_bin_module():
 _have_equal_features = _catboost._have_equal_features
 SPARSE_MATRIX_TYPES = _catboost.SPARSE_MATRIX_TYPES
 
+_CATBOOST_SKLEARN_COMPAT_TAGS = {
+    'non_deterministic': False,
+    'requires_positive_X': False,
+    'requires_positive_y': False,
+    'X_types': ['2darray'],
+    'poor_score': False,
+    'no_validation': False,
+    'multioutput': False,
+    "allow_nan": False,
```

Comment:
this should be true

Reply:
Thanks, I missed this [great documentation on sklearn estimator tags](https://github.com/scikit-learn/scikit-learn/blob/a847cdebd624a00b6fb21a4f941466727aeb81f2/doc/developers/develop.rst), so estimated it was boilerplate.